css/base.css

```diff
@@ -0,0 +1,49 @@
+/*text*/
+p {
+  font-family: 'Droid Serif', serif;
+  font-size: 1em;
+  color: #696969;
+  line-height: 1.5;
+}
+h2, h3, .sharebutton {
+  font-family: 'Abel', sans-serif;
+}
+.logo h1{
```

Comment:
Nitpick alert! Not dinging you for this, but just wanted to remind you to be consistent with your formatting :)
Space between the selector and opening brace is the convention - h1 {